src/commands/env/env-get.js

```diff
@@ -43,6 +62,16 @@ const createEnvGetCommand = (program) =>
   program
     .command('env:get')
     .argument('<name>', 'Environment variable name')
+    .addOption(
+      new Option('-c, --context <context>', 'Specify a deploy context')
+        .choices(['production', 'deploy-preview', 'branch-deploy', 'dev'])
+        .default('dev'),
+    )
+    .addOption(
+      new Option('-s, --scope <scope>', 'Specify a scope')
+        .choices(['builds', 'functions', 'post_processing', 'runtime', 'any'])
```

Comment:
One more functionality question (this will impact the docs) - can they filter for variables that have `all` set as the scope? (which is different than `any`, right?)

Reply:
I interpret `all` to mean "which env vars are available in builds and functions and post_processing and runtime scopes", vs `any` to mean "which env vars are available in builds or functions or post_processing or runtime scopes". I think `any` is more desirable when getting, and `all` more desirable when setting. 

Note that `any` doesn't exist on context, because context is on the value level (where there could be multiple things returned), whereas scope is on the env var level (where there can only be one thing returned).

Reply:
I agree with the distinction between `all` and `any`, so it makes sense for the default to be `any` here.

I thought of `all` as people might want to filter for only those that are set to `all` 🤔 But I actually don't know which use case is more valuable, let's ping @melanierichards for this one.

Reply:
> Note that any doesn't exist on context, because context is on the value level (where there could be multiple things returned), whereas scope is on the env var level (where there can only be one thing returned).

This makes sense to me!

Reply:
Both `any` and `all` seem to me to have their place.